osu.Android/OsuGameActivity.cs

```diff
@@ -14,10 +14,13 @@
 
 namespace osu.Android
 {
-    [Activity(Theme = "@android:style/Theme.NoTitleBar", MainLauncher = true, ScreenOrientation = ScreenOrientation.FullUser, SupportsPictureInPicture = false, ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize, HardwareAccelerated = false)]
+    [Activity(Theme = "@android:style/Theme.NoTitleBar", MainLauncher = true, ScreenOrientation = ScreenOrientation.FullUser, SupportsPictureInPicture = false, ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize, HardwareAccelerated = false, LaunchMode = LaunchMode.SingleInstance)]
     [IntentFilter(new[] { Intent.ActionDefault, Intent.ActionSend }, Categories = new[] { Intent.CategoryDefault }, DataPathPatterns = new[] { ".*\\.osz", ".*\\.osk" }, DataMimeType = "application/*")]
+    [IntentFilter(new[] { Intent.ActionView }, Categories = new[] { Intent.CategoryBrowsable, Intent.CategoryDefault }, DataScheme = osu_url_scheme)]
```

Comment:
iOS [also routes `osump://` scheme links](https://github.com/ppy/osu/blob/3087aba5bc359c5a23075882d89578c3f2fd45aa/osu.iOS/Info.plist#L120-L123). While the game [can't handle them yet](https://github.com/ppy/osu/blob/fa94703b9b663e443896a3971bfc2c2102350b93/osu.Game/OsuGame.cs#L270-L278), might as well add them while we're here for full parity.

Reply:
whoops missed that one, nice catch